### ISSUE TITLE
DeepOPF: A Feasibility-Optimized Deep Neural Network Approach for AC Optimal Power Flow Problems. (arXiv:2007.01002v3 [eess.SY] UPDATED)

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#10417]
DeepOPF: A Feasibility-Optimized Deep Neural Network Approach for AC Optimal Power Flow Problems. (arXiv:2007.01002v3 [eess.SY] UPDATED)<br>
https://ift.tt/38NVleZ<br>
<p>We develop a Deep Neural Network (DNN) approach, named DeepOPF, for solving alternative current optimal power flow (AC-OPF) problems. A key difficulty for applying machine learning techniques for solving AC-OPF problems lies in ensuring that the obtained solutions respect the equality and inequality physical and operational constraints. Generalized a 2-stage procedure proposed in [1], [2], DeepOPF first trains a DNN model to predict a set of independent operating variables and then directly compute the remaining dependable ones by solving the AC power flow equations. Such an approach not only preserves the power-flow balance equality constraints, but also reduces the number of variables to predict by the DNN, subsequently cutting down the number of neurons and training data needed. DeepOPF then employs a penalty approach with a zero-order gradient estimation technique in the training process to preserve the remaining inequality constraints. As another contribution, we drive a condition for tuning the size of the DNN according to the desired approximation accuracy, which measures the DNN generalization capability. It provides theoretical justification for using DNN to solve AC-OPF problem. Simulation results of IEEE 30/118/300-bus and a synthetic 2000-bus test cases show that DeepOPF can speed up the computing time by up to two orders of magnitude as compared to a state-of-the-art solver, at the expense of $&lt;$0.1% cost difference.</p>
<br>
<br>
via eess.SY updates on arXiv.org http://arxiv.org/<br>
